include/tokens.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef TOKENS_HPP_
+#define TOKENS_HPP_
+#include <json_fwd.hpp>
+#include "runebound_fwd.hpp"
+
+namespace runebound {
+namespace token {
+
+void to_json(nlohmann::json &json, const Token &token);
+void from_json(const nlohmann::json &json, Token &token);
+
+enum class TokenType { BATTLE, ADVENTURE, PLOT };
+
+struct Token {
+private:
+    TokenType m_type;
+
+public:
+    Token() = default;

```

Comment:
Exclicit

Reply:
Nope, it's an empty constructor so `explicit` will have no effect.